web-components/src/components/cards/ArticleCard/ArticleCard.tsx

```diff
@@ -56,7 +56,7 @@ export default function ArticleCard({
       elevation={1}
       draftText={draftText}
     >
-      {play && <PlayButton />}
+      {play && <PlayPauseButton playing={false} />}
```

Comment:
I think it makes it a bit more clear that this is just a play button. 

Although, coming back to our conversation about this component, maybe a `VideoControls` component that has the playing state instead of the `PlayPauseButton` makes more sense. We can then keep the `Play` and `Pause` buttons separated so we can use the play button independently. If we want to avoid repeating styles between Play/Pause we could keep the `PlayPauseButton` (`MediaButton` or similar name) and pass down a prop specifying the button type, i.e: `<MediaButton type='play' />` or `<MediaButton type='pause' />`

Reply:
I'm fine with a MediaButton too, then whether to use a boolean `playing` or string `type` as prop doesn't really matter to me. I just want to avoid having repeated styles at different places.